src/loader/bgw_message_queue.c

```diff
@@ -0,0 +1,377 @@
+#include <postgres.h>
```

Comment:
We should link to our source for this queue, if we have one. One such source is http://opendatastructures.org/ods-python/2_3_ArrayQueue_Array_Based_.html  no proof though.